crates/uv/src/main.rs

```diff
@@ -651,6 +651,10 @@ struct VenvArgs {
     #[clap(default_value = ".venv")]
     name: PathBuf,
 
+    /// The prompt used in shells.
```

Comment:
Can we add a description of what the promot is and what it is used for? 

It also seems that the prompt value `.` has a special meaning that it uses the current working directory which isn't clear otherwise. It also seems to default to `.` if you use `.venv` which might be surprising to users. 

Is defaulting to `.` to use the current directory name the same as in pip? I wonder if we could come up with a design that uses less "magical" values

Reply:
I've expanded the CLI docs with the default behavior which depends on the venv name and also with the list of possible values.